<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileCmpRuleWin</class>
 <widget class="QWidget" name="FileCmpRuleWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>231</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FileCmpRuleWin</string>
  </property>
  <property name="windowIcon">
   <iconset resource="RealCompare.qrc">
    <normaloff>:/Resources/img/main.png</normaloff>:/Resources/img/main.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Compare Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radioButtonDefault">
        <property name="text">
         <string>Ignore whitespace characters before line</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonIgnoreBackWhite">
        <property name="text">
         <string>Ignore whitespace characters At back of the line(such as python)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonIgnoreAllWhite">
        <property name="text">
         <string>Ignore all whitespace characters</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Match Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="checkBoxBlankMatch">
        <property name="text">
         <string>Blank lines participate in matching</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Identify matching rates for rows that are equal</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxEqualRato">
          <item>
           <property name="text">
            <string>Match &gt;= 50%</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Match &gt;= 70%</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Match &gt;= 90%</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="RealCompare.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>FileCmpRuleWin</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>FileCmpRuleWin</receiver>
   <slot>slot_apply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_apply()</slot>
 </slots>
</ui>
